packages/lib/modules/pool/actions/create/modal/useInitializePoolStep.tsx

```diff
@@ -54,20 +53,21 @@ export function useInitializePoolStep({
     initPoolInput,
   })
 
-  const gasEstimationMeta = sentryMetaForWagmiSimulation('Error in initialze LBP gas estimation', {
-    buildCallQueryData: buildCallDataQuery.data,
-    tenderlyUrl: buildTenderlyUrl(buildCallDataQuery.data),
-  })
-
-  const isComplete = () => !!isPoolInitialized || isTransactionSuccess(transaction)
+  const gasEstimationMeta = sentryMetaForWagmiSimulation(
+    'Error in initialize pool gas estimation',
+    {
+      buildCallQueryData: buildCallDataQuery.data,
+      tenderlyUrl: buildTenderlyUrl(buildCallDataQuery.data),
+    }
+  )
 
   return useMemo(
     () => ({
       id: initializePoolStepId,
       stepType: 'initializePool',
       labels,
       transaction,
-      isComplete,
+      isComplete: () => !!isPoolInitialized || isTransactionSuccess(transaction),
```

Comment:
isn't `isPoolInitialized` always a boolean?

Reply:
it can be `undefined` too